.mergify.yml

```diff
@@ -0,0 +1,18 @@
+# See https://doc.mergify.io
+
+pull_request_rules:
+- name: auto_merge
+  conditions: 
+    - -title~=(WIP|wip)
+    - -merged
+    - -closed
+    - -locked
+    - -label=blocked
+    - "#approved-reviews-by>=1"
+    - "#changes-requested-reviews-by=0"
+    - status-success=continuous-integration/travis-ci/pr
```

Comment:
This is not needed. CodeBuild is the "official" one (and occasionally travis will pend for a long time to wait for a box).